apps/evm/src/pages/Market/LidoPoolMarket/index.tsx

```diff
@@ -1,27 +1,22 @@
 import { useParams } from 'react-router-dom';
 
 import { useGetChainMetadata } from 'hooks/useGetChainMetadata';
-import { useIsFeatureEnabled } from 'hooks/useIsFeatureEnabled';
 
 import MarketLoader from 'containers/MarketLoader';
 import { Market } from '../Market';
-import MarketOld from '../MarketOld';
 
-const StakedEthPoolMarket: React.FC = () => {
+const LidoPoolMarket: React.FC = () => {
```

Comment:
Shouldn't we keep this as StakedEthPoolMarket? It's actually the file name that is outdated

Reply:
Actually, it seems this file isn't used at all. You're right that the name was changed to `StakedEthPoolMarket`, but we already have a file for it. I'll just remove `LidoPoolMarket`